lms/djangoapps/courseware/module_render.py

```diff
@@ -1169,6 +1170,8 @@ def hash_resource(resource):
     return md5.hexdigest()
 
 
+@api_view(['GET'])
+@view_auth_classes(is_authenticated=True)
```

Comment:
FYI @robrap: This PR makes the XBlock view use auth classes instead of hardcoding to session auth -- please let me know if you have any questions or concerns.

Reply:
None here. FYI: @nasthagiri @douglashall